stdlib/public/SDK/Foundation/AffineTransform.swift

```diff
@@ -124,7 +124,7 @@ public struct AffineTransform : ReferenceConvertible, Hashable, CustomStringConv
          [    0       0    1 ]
      */
     public init(rotationByDegrees angle: CGFloat) {
-        let α = Double(angle) * M_PI / 180.0
+        let α = Double(angle) * .pi / 180.0
         self.init(rotationByRadians: CGFloat(α))
```

Comment:
Oh. right! `CGFloat` conforms to the same protocol. Makes perfect sense. Thanks!